security-keycloak-authorization-quickstart/README.md

```diff
@@ -46,13 +46,15 @@ Launch the Maven build on the checked out sources of this demo:
 
 > :warning: **NOTE**: Do not start the Keycloak server when you run the application in a dev mode - `Dev Services for Keycloak` will launch a container for you.
 
-To start a Keycloak Server you can use Docker and just run the following command:
+To start a Keycloak Server you can use Docker and just run the following command in the root directory of this quickstart:
 
 ```bash
-docker run --name keycloak -e DB_VENDOR=H2 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin -p 8180:8080 -p 8543:8443 quay.io/keycloak/keycloak:{keycloak version}
+docker run --name keycloak -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin -p 8543:8443 -v "$(pwd)"/config/keycloak-keystore.jks:/etc/keycloak-keystore.jks quay.io/keycloak/keycloak:{keycloak.version} start  --hostname-strict=false --https-key-store-file=/etc/keycloak-keystore.jks
```

Comment:
Would make it easier to just run `start-dev`? In general, we are using this command for dev/testing purposes so that you don't need to worry about production-ready settings.